packages/framer-motion/package.json

```diff
@@ -101,7 +101,7 @@
         },
         {
             "path": "./dist/size-rollup-animate.js",
-            "maxSize": "10 kB"
+            "maxSize": "15 kB"
```

Comment:
Adding renderer is expensive but could be less with a later refactor, would be a fair bit of work tho and this is mostly shared with `motion` component